releasenotes/notes/jmxfetch-support-uds-dff1750ddba3b575.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
```

Comment:
You might want to check if we add release notes when JMXFetch versions are bumped (in this case to `0.40.0`), my recollection is we do add these notes.